wgengine/magicsock/magicsock.go

```diff
@@ -1748,27 +1875,25 @@ func (c *Conn) receiveIP(b []byte, ipp netip.AddrPort, cache *ippEndpointCache,
 	return ep, true
 }
 
-// receiveDERP reads a packet from c.derpRecvCh into b and returns the associated endpoint.
-// It is called by wireguard-go.
-//
-// If the packet was a disco message or the peer endpoint wasn't
-// found, the returned error is errLoopAgain.
-func (c *connBind) receiveDERP(b []byte) (n int, ep conn.Endpoint, err error) {
+func (c *connBind) receiveMultipleDERP(buffs [][]byte, sizes []int, eps []conn.Endpoint) (int, error) {
```

Comment:
this is only receiving 1 packet at a time, is that ok?

Reply:
I ended up renaming the Receive fns back to their original names. The `multiple` part originated from when we had both single packet and multi packet Receive fns prior to flattening the wg-go conn.Bind API.

There is also more use of the previous names in health/stats/docs that it just made sense to rename back.